TLPhotoPicker/Classes/TLAssetsCollection.swift

```diff
@@ -420,10 +420,14 @@ extension UIImage {
         case .up:
             return self
         default:
-            UIGraphicsBeginImageContextWithOptions(size, false, scale)
-            draw(in: CGRect(origin: .zero, size: size))
-            let result = UIGraphicsGetImageFromCurrentImageContext()
-            UIGraphicsEndImageContext()
+            var format = UIGraphicsImageRendererFormat()
```

Comment:
this looks like it is only used in this public function which we currently do not use in the PG app  public func tempCopyMediaFile(videoRequestOptions: PHVideoRequestOptions? = nil,
                                  imageRequestOptions: PHImageRequestOptions? = nil,
                                  livePhotoRequestOptions: PHLivePhotoRequestOptions? = nil,
                                  exportPreset: String = AVAssetExportPresetHighestQuality,
                                  convertLivePhotosToJPG: Bool = false,
                                  progressBlock:((Double) -> Void)? = nil,
                                  completionBlock:@escaping ((URL,String) -> Void)) -> PHImageRequestID? {